python/jupyterlab-chat/jupyterlab_chat/ychat.py

```diff
@@ -183,11 +185,14 @@ def get(self) -> str:
         Returns the contents of the document.
         :return: Document's contents in JSON.
         """
-        return json.dumps({
-            "messages": self.get_messages(),
-            "users": self.get_users(),
-            "metadata": self.get_metadata()
-        })
+        return json.dumps(
+            {
+                "messages": self.get_messages(),
+                "users": self.get_users(),
+                "metadata": self.get_metadata()
+            },
+            indent=2
+        )
```

Comment:
Just curious: does this change make the `*.chat` files more human-readable from a command-line editor? Right now `*.chat` files are a compressed one-line JSON string.

Reply:
Yes, it is the intention to make it more readable for any text editor.